samples/quickstart.js

```diff
@@ -1,38 +1,36 @@
-// Copyright 2020 Google LLC
-//
+// Copyright 2020, Google LLC.
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     https://www.apache.org/licenses/LICENSE-2.0
+//    http://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
 
 'use strict';
 
-async function quickstart() {
-  // parent = 'projects/my-project' // Project to list dashboards for.
+/**
+ * Instantiate a Media Translation client.
+ */
+async function main() {
   // [START media_translation_quickstart]
-  // Imports the Google Cloud client library
   const {
     SpeechTranslationServiceClient,
   } = require('@google-cloud/media-translation');
 
-  // Creates a client
-  const translate = new SpeechTranslationServiceClient();
-
-  // parent = 'projects/my-project', // Project to list dashboards for.
-
-  // TODO: add an actual sample.
-  console.info(translate);
-
+  async function quickstart() {
+    const client = new SpeechTranslationServiceClient();
+    console.log(client);
```

Comment:
we're not actually doing anything async in this sample yet, is there a minimal example we could toss in here?

Reply:
Good point. I copied the translate_from_file.js into the quickstart.